app/scripts/templates/sign_in_unblock.mustache

```diff
@@ -0,0 +1,43 @@
+<div id="main-content" class="card">
+  {{#error}}
+    <header>
+      <h1 id="fxa-signin-unblock-header">{{#t}}Authorize this sign-in{{/t}}</h1>
```

Comment:
Good point, I'll change this to something like "problem sending unblock email" because that's how this error condition occurs.